examples/04_pong/main.rs

```diff
@@ -335,6 +338,20 @@ impl State for Pong {
                 bounce_sfx,
                 score_sfx,
             });
+            // Sets the cursor hidden when the mouse is hovering the pong window.
+            if let Some(mut window_events) = engine
+                .world
+                .res
+                .try_fetch_mut::<EventHandler<WindowModifierEvent>>(0)
+            {
+                window_events.write_single(WindowModifierEvent {
```

Comment:
This section ends up looking a bit verbose with use of shrev.  This might look a bit cleaner if we used a resource like

```rust
pub struct WindowCommands {
    inner: SmallVec<[Box<FnOnce(&mut winit::Window)>; 5]>
}

impl WindowCommands {
    pub fn send_command(&mut self, command: Box<FnOnce(&mut winit::Window)>) {
        self.inner.push(command);
    }
```

Then just drain that vec in the renderer.

Reply:
To put it another way: shrev is really good if you might want more than one ReaderId, if you only have one ReaderId though it offers little benefit and mostly just complicates the implementation.